### PR TITLE
Join event

### PR DESCRIPTION
**Pull request by ConnorFoody**
#10 maybe a little of #22?

### CODE REVIEW

File: MeNext/App.xaml.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public App()
 
 		protected override void OnStart()
 		{
-			// Handle when your app starts
```
Comment (ConnorFoody) [inline MeNext/App.xaml.cs:18]: Why get rid of this line?

File: MeNext/Layout/HostEvent.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	{
+		public HostEvent()
+		{
+			var layout = new StackLayout
+			{
+				Children = {
+					new Label { Text = "Host Event!\nVarious permissions in a list\n" }
+				}
+			};
+			var permissions = new List<string>();
```
Comment (ConnorFoody) [inline MeNext/Layout/HostEvent.cs:17]: We should probably convert this to a type. We probably want descriptions, plain text names, and server permission names.

File: MeNext/Layout/PlaceholderScreen.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿
+using System;
```
Comment (ConnorFoody) [inline MeNext/Layout/PlaceholderScreen.cs:2]: get rid of this change

File: MeNext/Layout/MainPage.cs [hunk 4]
```diff
@@ -24,18 +24,22 @@ public MainPage()
 
 			var homeScreen = new NavigationPage(new HomeScreen());
 			homeScreen.Title = "Home";
+			//homeScreen.Icon = "homeScreenIcon.png";  If we make this icon
```
Comment (ConnorFoody) [inline MeNext/Layout/MainPage.cs:27]: Does this work? If so any reason not to leave it in?

@samsaksfithian @daniel-centore thoughts?

File: MeNext/Layout/JoinEvent.cs [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+		public JoinEvent()
+		{
+			var layout = new StackLayout
+			{
+				Children = {
+					new Label { Text = "Join Event" }
+				}
+
+			};
+			layout.Children.Add(new Entry { Text = "Event Name/Id Code" });
```
Comment (ConnorFoody) [inline MeNext/Layout/JoinEvent.cs:18]: Make this text disappear when you click on it. We can do this later though

**Review by ConnorFoody — COMMENTED** [review]
I have some issues with the UI that we can talk about at meeting today. 

Review is mostly little nits

File: MeNext/Layout/MainPage.cs (hunk 4 above)
Comment (daniel-centore) [reply MeNext/Layout/MainPage.cs:27]: I see no reason not to leave it in. Just leave it commented. The goal is to eventually have icons, even if super low priority.

File: MeNext/Layout/JoinEvent.cs (hunk 5 above)
Comment (daniel-centore) [reply MeNext/Layout/JoinEvent.cs:18]: Literally all you need to do is change `Text` to `Placeholder`

**Pull request merged**